indra-common/src/main/kotlin/net/kyori/indra/IndraExtension.kt

```diff
@@ -123,6 +125,11 @@ open class IndraExtension @Inject constructor(objects: ObjectFactory) {
     url = "https://opensource.org/licenses/MIT"
   ))
 
+  fun jenkins(url: String) = this.continuousIntegration.set(ContinuousIntegration(
```

Comment:
should add GH actions and GitLab CI as well -- and put add it as one of the things that can be configured by `github` and `gitlab`